src/com/palmergames/bukkit/towny/command/TownCommand.java

```diff
@@ -498,6 +499,18 @@ public List<String> onTabComplete(CommandSender sender, Command command, String
 						default:
 							return Collections.emptyList();
 					}
+				case "trusttown":
+					switch (args.length) {
+						case 2:
+							return NameUtil.filterByStart(Arrays.asList("add", "remove", "list"), args[1]);
+						case 3:
+							if (args[1].equalsIgnoreCase("add") || args[1].equalsIgnoreCase("remove"))

```

Comment:
remove should only list towns which are already trusted, add should only list towns which aren't already trusted.

Reply:
Implemented as requested using other similar use-cases as an example.